jupyter_server/serverapp.py

```diff
@@ -1435,38 +1433,13 @@ def template_file_path(self):
         help="""If True, display controls to shut down the Jupyter server, such as menu items or buttons.""",
     )
 
-    # REMOVE in VERSION 2.0
```

Comment:
@Zsailer I believe removing this is what is causing the nbclassic downstream test to fail.  Do we want to keep this in for another major release?

Reply:
Okay, punted to 3.0

Reply:
Okay, so the actual issue occurs when we change the default value from `LargeFileManager` -> `AsyncLargeFileManager`, which was the main idea of this change.  😅

Reply:
Ah, nbclassic was assuming synchronous contents manager:

```
  tests/test_notebookapp.py::test_tree_handler
    /home/runner/test_venv/downstream_test/nbclassic/nbclassic/tree/handlers.py:53: RuntimeWarning: coroutine 'AsyncFileContentsManager.dir_exists' was never awaited
      if cm.dir_exists(path=path):
    Enable tracemalloc to get traceback where the object was allocated.
    See https://docs.pytest.org/en/stable/how-to/capture-warnings.html#resource-warnings for more info.
  
  tests/test_notebookapp.py::test_tree_handler
    /home/runner/test_venv/downstream_test/nbclassic/nbclassic/tree/handlers.py:54: RuntimeWarning: coroutine 'AsyncFileContentsManager.is_hidden' was never awaited
```

We need to update it regardless.